src/node.cc

```diff
@@ -2363,7 +2403,7 @@ static void OnFatalError(const char* location, const char* message) {
 
 NO_RETURN void FatalError(const char* location, const char* message) {
   OnFatalError(location, message);
-  // to suppress compiler warning
+  // To suppress compiler warning
```

Comment:
If you're making an unrelated stylistic change, then at least also punctuate the comment. :-)

(Ditto on line 2459.)